AppleMusic/Source/View/RadioView/StationView.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  StationView.swift
+//  AppleMusic
+//
+//  Created by Artem Muho on 20.02.2022.
+//
+
+import SwiftUI
+
+struct StationView: View {
+    @State var columnsData = StationModel.getColumns()
+    
+    @State var columns = [
+        GridItem(.flexible())
+    ]
+    
+    var body: some View {
+        LazyVGrid(columns: columns, alignment: .leading) {
+            Text("Станции")
+                .font(.title2)
+                .bold()
+                .offset(x: 15)
+            ForEach(columnsData, id: \.self) { column in
+                HStack(alignment: .center) {
+                    column.image
+                        .resizable()
+                        .frame(width: 130, height: 130)
+                        .cornerRadius(6)
+                    VStack(alignment: .leading) {
+                        Text(column.title)
+                            .font(.title2)
+                        Text(column.discription)
+                            .font(.subheadline)
+                            .foregroundColor(.gray)
+                        Spacer()
+                        Divider()
+                    }
+                    .frame(height: 130)
+                }
+                .padding()
+            }
+            .frame(height: 130)
```

Comment:
У тебя ячейка уже ограничена 130, поэтому повторно не надо ограничивать.
```suggestion
                .padding(.horizontal)
            }
```